pkg/apic/definitions.go

```diff
@@ -104,6 +104,7 @@ type ServiceBody struct {
 	ImageContentType  string
 	CreatedBy         string
 	ResourceType      string
+	AltRevisionPrefix    string
```

Comment:
There is a Prettier code formatter extension (if you are using VSCode) that will auto align things in structs, fix inconsistent whitespacing (which I see elsewhere) that we all have installed.

Reply:
@dfeldick can you send me what your are using and the configuration/setup?

Reply:
👍  code has been formatted